lms/djangoapps/course_structure_api/v0/api.py

```diff
@@ -0,0 +1,94 @@
+"""
+API implementation of the Course Structure API for Python code.
+
+Note: The course list and course detail functionality isn't currently supported here because of the tricky interactions between DRF and the code.
+Most of that information is available by accessing the course objects directly.
+"""
+
+from course_structure_api.v0 import serializers
+from course_structure_api.v0.errors import CourseNotFoundError, CourseStructureNotAvailableError
+from openedx.core.djangoapps.content.course_structures import models, tasks
+from courseware import courses
+
+
+def _retrieve_course(course_key):
```

Comment:
@wedaly if you are the owner of the course structure API now, you might be interested in reviewing this refactoring.

Also, when will this API be bumped to v1?